Add docs for all fields in events.proto

alexanderkarpetc opened the pull request:
Now all fields have the same docs and format. 
Example:

```
    // The same as the bookId.
    InventoryId inventory_id = 1;

    // When inventory was created.
    google.protobuf.Timestamp when_created = 2;
```

alexanderkarpetc committed:
Add docs for all fields in events.proto

Dmytry-Dyachenko reviewed (approved):
LGTM

Dmytry-Dyachenko commented on model/src/main/proto/javaclasses/exlibris/c/events.proto:
```diff
@@ -36,12 +36,12 @@ import "google/protobuf/timestamp.proto";
-//
 message InventoryCreated {
 
     // The same as the bookId.
```
Change that doc. There just ID of inventory. Another information must be in `InventoryId`.

Dmytry-Dyachenko committed:
Change case of RFID in comments  

````````````````
From: // Optional field: exists when book has Rfid mark.
to:      // Optional field: exists when book has RFID mark.
````````````````

alexanderkarpetc committed:
Rename RFID -> Rfid

alexanderkarpetc committed:
Merge remote-tracking branch 'origin/events_docs_for_fields' into events_docs_for_fields

alexanderkarpetc committed:
Grammar and semantics fixes of docs

alexanderkarpetc committed:
Rename RFID in commands.proto -> Rfid
